File: py110/small_problems_110_119/easy2/2.py
# https://launchschool.com/exercises/9db45ac4

'''
Questions:


Input: 2 lists
Output: set
Explicit:
Implicit:


Pseudocode:

'''

def union(lst1, lst2):
    return set(lst1) | set(lst2)

print(union([1, 3, 5], [3, 6, 9]) == {1, 3, 5, 6, 9}) # True

File: js210/simpleProblems/easy4/2.js
// https://launchschool.com/exercises/8c72838d

'use strict';

function union(arr1, arr2) {
  return [...new Set([...arr1, ...arr2])];
}

console.log(union([1, 3, 5], [3, 6, 9]));    // [1, 3, 5, 6, 9]
